services/kafka/Dockerfile

```diff
@@ -28,8 +28,8 @@ ENV KAFKA_RELEASE_ARCHIVE kafka_2.11-0.9.0.0.tgz
 # Install Kafka to /kafka
 
 WORKDIR /tmp
-RUN wget --no-verbose http://www.us.apache.org/dist/kafka/0.9.0.0/${KAFKA_RELEASE_ARCHIVE} && \
-    wget --no-verbose https://dist.apache.org/repos/dist/release/kafka/0.9.0.0/${KAFKA_RELEASE_ARCHIVE}.md5  && \
+RUN wget --no-verbose http://www.us.apache.org/dist/kafka/0.9.0.1/${KAFKA_RELEASE_ARCHIVE} && \
+    wget --no-verbose https://dist.apache.org/repos/dist/release/kafka/0.9.0.1/${KAFKA_RELEASE_ARCHIVE}.md5  && \
```

Comment:
While we're at it: How about adding an ENV var above which contains the Kafka version like

`KAFKA_VERSION 0.9.0.1` and use it both in the `RELEASE_ARCHIVE` and the urls?